CODEOWNERS

```diff
@@ -179,6 +179,8 @@
 /specification/purview/data-plane @mikekistler @Azure/api-stewardship-board
 
 # PRLabel: %PostgreSQL
+/specification/postgresql/** @microsoft/azurepostgresql
```

Comment:
You cannot use this team in codeowners as it will not work for as it isn't a team in the Azure org with write permissions to the repo.

Reply:
I am working with owner of this group to get the desired permissions added

Reply:
I'm the owner of this repo and we cannot grant permissions to a team outside of the Azure github org. Github doesn't support that.

Reply:
Does it mean that we need to create a team something like @Azure\azurepostgesql and get write permissions for this repo added to this team?

Reply:
Yes if you wish to use a team in this code owners file then yes you will need to do that. You should get an azure-sdk-write-postgresql team created, see https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/49/Request-Access-to-Azure-SDK-repos

Reply:
Tagging @pradeepkintali